owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/GetRemoteShareesOperation.kt

```diff
@@ -173,12 +181,6 @@ class GetRemoteShareesOperation
         private const val VALUE_ITEM_TYPE = "file"         //  to get the server search for users / groups
 
         // JSON Node names
-        private const val NODE_OCS = "ocs"
-        private const val NODE_DATA = "data"
-        private const val NODE_EXACT = "exact"
-        private const val NODE_USERS = "users"
-        private const val NODE_GROUPS = "groups"
-        private const val NODE_REMOTES = "remotes"
         const val NODE_VALUE = "value"
         const val PROPERTY_LABEL = "label"
         const val PROPERTY_SHARE_TYPE = "shareType"
```

Comment:
Some constants are not used anywhere, could you take a look?

Reply:
Those properties are not used anymore, you parse it directly using moshi. I think you can remove them.